help/destinations/catalog/adobe/marketo-engage-person-sync.md

```diff
@@ -0,0 +1,120 @@
+
+>[!IMPORTANT]
+>
+>The user setting up the destination must have [Edit Person](https://experienceleague.adobe.com/en/docs/marketo/using/product-docs/administration/users-and-roles/descriptions-of-role-permissions#access-database) permission in the Marketo instance and partition.
+
+![Connect to Destination](../../assets/catalog/adobe/marketo-engage-person-sync/connect-to-destination.png)
+
+Once you have selected your instance, you will also need to select the Lead Partition which you want the configuration to integrate with. A [Lead Partition](https://experienceleague.adobe.com/en/docs/marketo/using/product-docs/administration/workspaces-and-person-partitions/understanding-workspaces-and-person-partitions) is a concept in Marketo Engage used to separate lead records by business concern, such as a brand or a sales region. If your Marketo subscription does not have the Workspaces and Partitions feature, or if no additional partitions have been created in your subscription, then only the Default partition will be available. A single configuration can only update lead records which exist in its configured partition.
+
+>[!IMPORTANT]
+> 
+>After a segment has been activated to the Marketo destination for the first time, backfilling profiles that already existed in the segment prior to Marketo destination activation can take [!UICONTROL up to 24 hours]. Going forward, any time profiles are added to the segment, they’ll be added to Marketo immediately.
+
+### Deduplication Fields {#deduplication-fields}
+
+When sending updates to Marketo engage, records are selected based on the selected partition and one or two user-selected fields. If your destination is configured with the North America partition, and has Email Address and Company Name configured as deduplication fields, then all three fields must match to apply changes to an existing record. For example:
+
+- Destination is configured with the North America partition
+- Person with Email <test@example.com> and Company name Example Inc. In AEP matches the destination audience
+- Unless a record with those values already exists in the North America partition in Marketo, a new lead record will be created
+
+If no matching lead record is found, a new record will be created.
+
+![Destination Details](../../assets/catalog/adobe/marketo-engage-person-sync/destination-details.png)
+
+## Activate Audiences {#activate-audiences}
+
+>[!IMPORTANT]
+> 
+>* To activate data, you need the **[!UICONTROL View Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+In the Activate Audiences step, you will be able to select from any person audiences which are visible to you.
```

Comment:
please add a blank line between the note and this line

also, let's link out to the generic activate audiences doc like in the template
https://experienceleague.adobe.com/en/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template#activate

btw, the template can be downloaded at the top of page here if you haven't done so already
https://experienceleague.adobe.com/en/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template